ignite/pkg/cosmosanalysis/app/app.go

```diff
@@ -140,6 +145,32 @@ func FindRegisteredModules(chainRoot string) (modules []string, err error) {
 	return modules, nil
 }
 
+// CheckAppWiring check if the app wiring exists finding the `appconfig.Compose` method call.
+func CheckAppWiring(chainRoot string) (bool, error) {
+	// Assumption: modules are registered in the app package
+	appFilePath, err := cosmosanalysis.FindAppFilePath(chainRoot)
+	if err != nil {
+		return false, err
+	}
+	// The directory where the app file is located.
+	// This is required to resolve references within the app package.
+	appDir := filepath.Dir(appFilePath)
+
+	appPkg, _, err := xast.ParseDir(appDir)
+	if err != nil {
+		return false, err
+	}
+
+	// Loop on package's files
+	for _, f := range appPkg.Files {
+		exists := goanalysis.FuncVarExists(f, appWiringImport, appWiringCallMethod)
```

Comment:
Shouldn't this call be checking for `appconfig.Compose`?

Reply:
I think there are other cases we might want to detect a part from the `appconfig.Compose` one:
- `appConfig = appconfig.Compose(...)`
- `appConfig = appconfig.LoadYAML(...)`
- `appConfig = appconfig.LoadJSON(...)`
- `appConfig = depinject.Configs(...)`

Reply:
Sorry, I changed this based on the Ilker comment. I will update the description. I think it can solve this `appconfig` cases:

https://github.com/ignite/cli/pull/3434#pullrequestreview-1299033916